keanu-project/src/main/java/io/improbable/keanu/vertices/intgr/probabilistic/PoissonVertex.java

```diff
@@ -1,5 +1,6 @@
 package io.improbable.keanu.vertices.intgr.probabilistic;
 
+import annotation.ExportVertexToPythonBindings;
```

Comment:
The annotation package should be `io.improbable.keanu.annotation` or maybe `annotation` should be `python.generation` or something...

Reply:
How about `io.improbable.keanu.annotation` for annotations and `io.improbable.keanu.annotation.processor` for processors?